templates/frontend/pages/catalogSeriesIndex.tpl

```diff
@@ -0,0 +1,66 @@
+{**
+ * templates/frontend/pages/catalogSeries.tpl
+ *
+ * Copyright (c) 2014-2017 Simon Fraser University Library
+ * Copyright (c) 2003-2017 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @brief Display the page to view books in a series in the catalog.
+ *
+ * @uses $series Series Current series being viewed
+ * @uses $publishedMonographs array List of published monographs in this series
+ * @uses $featuredMonographIds array List of featured monograph IDs in this series
+ * @uses $newReleasesMonographs array List of new monographs in this series
+ *}
+{include file="frontend/components/header.tpl" pageTitle="plugins.block.browse.series"}
+
+<div class="page page_catalog_seriesIndex">
+
+	{* Breadcrumb *}
+	{include file="frontend/components/breadcrumbs_catalog.tpl" type="series" currentTitleKey="plugins.block.browse.series"}
+
+	<h2>
+		{translate key="plugins.block.browse.series"}
+	</h2>
+
+	{* Index with series *}
+	<div class="seriesIndex">
+	<ul>
+
+		{* Series *}
+		{if $browseSeriesFactory && $browseSeriesFactory->getCount()}
+
+		{iterate from=browseSeriesFactory item=browseSeriesItem}				
+		<li>
+		
+		<div class="imageDescription">		
+
+	{* Image and description *}
+
+			<div class="cover"> <a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="series" path=$browseSeriesItem->getPath()|escape}">
+			<img src="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="thumbnail" type="series" id=$browseSeriesItem->getId()|escape}" alt="{$browseSeriesItem->getLocalizedFullTitle()|escape}" /></a>
+			</div>
+
+			<div class="metadata">				
+			<h3><a href="{url router=$smarty.const.ROUTE_PAGE page="catalog" op="series" path=$browseSeriesItem->getPath()|escape}"> {$browseSeriesItem->getLocalizedFullTitle()|escape}</a></h3>
```

Comment:
Change `h3` to `h2` to reflect the page structure.

Reply:
in that case the h2 at the start of the page needs to become a h1
I understand the reasoning that the larger heading at a page is h1,
but from a site perspective we could have a different view.
I do as suggested.

Reply:
Yes, good catch. The page title must be in `<h1>`.